packages/build/src/plugins_core/functions/index.js

```diff
@@ -38,10 +54,9 @@ const coreCommand = async function ({
   }
 
   try {
-    await zipFunctions(functionsSrc, functionsDist, {
-      externalModules,
-      useEsbuild: Boolean(childEnv.NETLIFY_EXPERIMENTAL_ESBUILD),
-    })
+    const zisiParameters = getZISIParameters({ childEnv, logs, netlifyConfig })
```

Comment:
[sand] This line might not need to be inside of the `try`/`catch`.

Reply:
Good call. Changed! ✅